index.js

```diff
@@ -0,0 +1,47 @@
+const actionModel = require('./data/models/actionModel');
+const projectModel = require('./data/models/projectModel');
+const express = require('express');
+const router = express.Router(); 
+// const {check, validationResult } = require('express-validator/check');
+const port = 3333;
+
+//server def and init
+const server = express();
+server.use(express.json());
+
+server.get('/project/:id', async (req,res) => {
+    try {
+        let id = parseInt(req.params.id);
```

Comment:
I don't think you need to parseInt the id, but in any case, you are doing it twice, which is completely unnecessary here.

Reply:
oddly enough I do need it for the findActionsbyProjectId, I had a database error where it wasn't returning any actions because it was looking for a string and not a number. (this took about 30 mins to find)